proto/imuachain/oracle/v1/two_phases.proto

```
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package imuachain.oracle.v1;
```

Buf (1.47.2)

Files with package "imuachain.oracle.v1" must be within a directory "imuachain/oracle/v1" relative to root but were in directory "proto/imuachain/oracle/v1".

(PACKAGE_DIRECTORY_MATCH)